### ISSUE TITLE
Plan migration from old ESI cluster to new ESI cluster

### ISSUE THREAD

**Issue by naved001** [CCI-MOC/ops-issues#1181]
Things we need to consider:
* VLAN Ranges 
* VLANs that are already allocated - what to do about those?
* DNS names. The current cluster is esi.massopen.cloud while the new one is moc-esi.massopen.cloud.
* How to migrate users?
* Migrate hardware - should be simple enough.

Keep in mind we want to keep the old cluster as a test or staging cluster. Feel free to add anything that we may need to consider.

**Comment by joachimweyl** [comment]
is the plan to rename esi.massopen.cloud to esi-test.massopen.cloud and then moc-esi.massopen.cloud to esi.massopen.cloud?

**Comment by naved001** [comment]
> is the plan to rename esi.massopen.cloud to esi-test.massopen.cloud and then moc-esi.massopen.cloud to esi.massopen.cloud?

Ideally yes, I don't know how much of a pain is it to change names of a cluster.

**Comment by joachimweyl** [comment]
@naved001 have we made any other decisions about the migration?

**Comment by naved001** [comment]
I talked to @tzumainn about this. 

We have already partitioned the VLANs, and new cluster will start with a smaller range, and as we migrate users from the old esi to new esi, we will update the VLAN ranges on each side.

The DNS names will remain unchanged.

We will move users in small batches, we don't have that many so should be simple.

Right now I need to fix the new esi cluster because after we moved the servers to a new location some networks changed and the cluster is unhappy. Once that's done we can migrate some hardware and users.
